db.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+$connect = mysqli_connect("localhost", "root", "root", "readme");
+if ($connect === false) {
+    die("Ошибка подключения: " . mysqli_connect_error());
+}
+// Запрос для типов постов
+function fetchPostTypes() {
+    $queryPostTypes = "SELECT id, name, icon_class from content_types";
+    return fetchData($queryPostTypes);
+}
+// Получаем шесть самых популярных постов и их авторов, а так же типы постов
+function fetchPopularPosts() {
+    $queryPopularPosts = "SELECT created_at as created_date,
+           title,
+           content as contain,
+           author,
+           users.login as user_name,
+           type_id,
+           views_count,
+           users.avatar_path as avatar
+    FROM posts
+        INNER JOIN users
+            ON user_id = users.id
+    ORDER BY views_count DESC
+    LIMIT 6";
+    return fetchData($queryPopularPosts);
```

Comment:
Как рекомендация по код стайлу -- делай отступы между логическими блоками.

```suggestion
    LIMIT 6";
    
    return fetchData($queryPopularPosts);
```

Reply:
И так везде)
Иначе читается плохо